io.github.hedge_dev.hedgemodmanager.yml

```diff
@@ -0,0 +1,53 @@
+app-id: io.github.hedge_dev.hedgemodmanager
+runtime: org.freedesktop.Platform
+runtime-version: "24.08"
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.dotnet8
+build-options:
+  prepend-path: /usr/lib/sdk/dotnet8/bin
+  append-ld-library-path: /usr/lib/sdk/dotnet8/lib
+  prepend-pkg-config-path: /usr/lib/sdk/dotnet8/lib/pkgconfig
+
+command: HedgeModManager.UI
+
+finish-args:
+  - --socket=x11
+  # UnleashedRecomp permissions
+  - --filesystem=~/.config/UnleashedRecomp:create
+  - --filesystem=~/.local/share/applications/UnleashedRecomp.desktop:ro
+  # Create permissions are required for accessing the mod loader, game and for patching game prefixes
+  # Steam permissions
+  - --filesystem=~/.steam/steam/steamapps:create
+  - --filesystem=~/.local/share/Steam/steamapps:create
+  # Heroic permissions
+  - --filesystem=~/Games/Heroic:create
+  # Common locations for game and mod files
+  - --filesystem=/media
+  - --filesystem=/run/media
+  - --filesystem=/mnt
```

Comment:
can you explain these filesystem permissions? What are these mainly for?

Reply:
Its already implemented it in my app, just not within Unleashed Recompiled yet, thought it would be a good idea to add it early. If you think I should remove it, and readd it later, I will.

And `UnleashedRecomp.desktop` is a file.